dags/bugzilla.py

```diff
@@ -0,0 +1,97 @@
+from airflow import DAG
+from airflow.operators.python import PythonOperator
+from airflow.models import Variable
+from datetime import datetime, timedelta
+import requests
+import time
+import logging
+
+from utils.tags import Tag
+
+docs = """
+### Bugzilla
+Runs daily scheduled jobs on Bugzilla.
+Tags bugs in Bugzilla for the product "Data Platform and Tools" and component "General" 
+with the whiteboard tag `[dataplatform]` if not already present. This will trigger a sync to Jira.
+
+#### Owner
+ascholtz@mozilla.com
+"""
+
+tags = [Tag.ImpactTier.tier_3]
+
+BUGZILLA_URL = "https://bugzilla.mozilla.org/rest"
+PRODUCT = "Data Platform and Tools"
+COMPONENT = "General"
+WHITEBOARD_TAG = "[dataplatform]"
+API_KEY_VAR = "BUGZILLA_API_KEY"
+
+default_args = {
+    "retries": 1,
+    "retry_delay": timedelta(minutes=30),
+    "owner": "ascholtz@mozilla.com",
+    "email": [
+        "telemetry-alerts@mozilla.com",
+        "ascholtz@mozilla.com",
+    ],
+    "depends_on_past": False,
+    "start_date": datetime(2025, 8, 3),
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 2,
+    "retry_delay": timedelta(minutes=30),
+}
+
+with DAG(
+    "bugzilla",
+    default_args=default_args,
+    schedule_interval="@daily",
+    tags=tags,
+    doc_md=docs,
+) as dag:
+
+    def fetch_and_tag_bugs():
+        api_key = Variable.get(API_KEY_VAR)
```

Comment:
I'll see if there is a way to have some "machine-user" credentials created. Until then using an API key associated with my personal account

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1981278